daemon/graphdriver/devmapper/deviceset.go

```diff
@@ -1083,6 +1083,126 @@ func determineDriverCapabilities(version string) error {
 	return nil
 }
 
+// Determine the major and minor number of loopback device
+func getDeviceMajorMinor(file *os.File) (uint64, uint64, error) {
+	stat, err := file.Stat()
+	if err != nil {
+		return 0, 0, err
+	}
+
+	Dev := stat.Sys().(*syscall.Stat_t).Rdev
+	majorNum := major(Dev)
+	minorNum := minor(Dev)
+
+	logrus.Debugf("devmapper: Major:Minor for device: %s is:%v:%v", file.Name(), majorNum, minorNum)
+	return majorNum, minorNum, nil
+}
+
+// Given a file which is backing file of a loop back device, find the
+// loopback device name and its major/minor number.
+func getLoopFileDeviceMajMin(filename string) (string, uint64, uint64, error) {
+	file, err := os.Open(filename)
+	if err != nil {
+		logrus.Debugf("devmapper: Failed to open file %s", filename)
+		return "", 0, 0, err
+	}
+
+	defer file.Close()
+	LoopbackDevice := devicemapper.FindLoopDeviceFor(file)
+	if LoopbackDevice == nil {
+		return "", 0, 0, fmt.Errorf("devmapper: Unable to find loopback mount for: %s", filename)
```

Comment:
I see many location in code where either we are not prefixing anything or just doing devmapper. I am not sure what's the convention now. Though it would be nice to make all of these uniform.